File: Dynamic-Programming/fibnacci/fibo-naive-recursive-approach.cpp
#include<bits/stdc++.h>
using namespace std;

/** Find nth fibnacci number(Exponential) **/

int fib(int n){
	if(n<=2){
		return 1;
	}
	else{
		return fib(n-1)+fib(n-2);
	}
}

int main(){
	// printing 10 fibonacci numbers
	for(int i = 1;i<=10;i++){
		cout<<fib(i);
		if(i!=10){
			cout<<' ';
		}
	}
	cout<<endl;
}

File: Dynamic-Programming/fibo-naive-recursive-approach.py
def fib(n):
	if n <= 2: return 1
	else : return fib(n-1)+fib(n-2)

#printing 10 fibonacci numbers
for i in range(1,11):
	if i == 10:
		print(fib(i))
		continue
	#print(fib(i),end=" ")
	print(f'{fib(i)} ',end= "")

